content/docs/pipelines/upgrade.md

```diff
@@ -111,6 +111,10 @@ performed the original deployment, represented in the deployment guide as
 cd ${KFAPP}
 ${KUBEFLOW_SRC}/scripts/upgrade_kfp.sh
 ```
+Or you can upgrade to a specific version of Pipeline by specifying the version ID. You can find the version ID in the Pipeline [release page](https://github.com/kubeflow/pipelines/releases). E.g. to upgrade to v0.1.12
```

Comment:
s/version ID/version tag

(twice)
(because we've used "tag" to describe the version ID in other parts of the docs)

Reply:
done